sklearn/model_selection/_search.py

```diff
@@ -918,15 +1005,32 @@ class RandomizedSearchCV(BaseSearchCV):
 
     Attributes
     ----------
-    grid_scores_ : list of named tuples
-        Contains scores for all parameter combinations in param_grid.
-        Each entry corresponds to one parameter setting.
-        Each named tuple has the attributes:
-
-            * ``parameters``, a dict of parameter settings
-            * ``mean_validation_score``, the mean score over the
-              cross-validation folds
-            * ``cv_validation_scores``, the list of scores for each fold
+    search_results_ : dict of numpy (masked) ndarrays
```

Comment:
> users who embark in the process of updating their projects to keep up with scikit-learn changes.

Okay with +3 I'm adding `grid_scores_` as a property function. Also I was wondering if we should store like we did before as computing them from `search_results_` (especially the params) is not very clean.